tests/basic/pool_function_test.go

```diff
@@ -84,6 +85,33 @@ var _ = Describe("{PoolExpandSmoky}", func() {
 		dash.VerifyFatal(resizeErr, nil, "Pool expansion does not result in error")
 		verifyPoolSizeEqualOrLargerThanExpected(poolIDToResize, targetSizeGiB)
 	})
+
+	// test resizing all pools at the same time and ensure only the first one makes progress
+	It("Select all pools and expand them concurrently. ", func() {
+		StartTorpedoTest("PoolExpandRejectConcurrent",
```

Comment:
Having StartTorpedoTest here will discard pushing all the logs prior to this like in JustBeforeEach to aetos. Logs will printed in jenkins but not in aetos

Reply:
Sorry what do you mean? 

I have a similar setup in https://github.com/portworx/torpedo/pull/1764, 
logs on aetos seem correct: 
https://aetos.pwx.purestorage.com/resultSet/testSetID/402969

Reply:
it works fine when there is no error or log.InfoD in JustBeforeEach, for example you can change lof.Infof to log.InfoD in JustBeforeEach, you will notice the log statement missing in aetos

Reply:
hmm this sounds like a bug that we should fix on the infra side..?

Reply:
no, StartTorpedoTest will create a testsetid in aetos for a test and pushes all logs to the testsetid. Here testsetid is created after JustBeforeEach executed and logging in JustBeforeEach will not be pushed to aetos as it doesn't have testsetId at that time

Reply:
Well in fact I think this is the correct behavior. JustBeforeEach contains test setup, and we do not want those logs to be present on aetos as part of the test per testsetID

Reply:
What happens if test fails before StartTorpedoTest is called?
That failure doesn't get logged to Aetos at all? 
If that is true, we can be in a situation where tests fail but we continue to report failures to Aetos.

Reply:
yes aetos will have the any log regard the test if any failure occurring in JustBeforeEach. Jenkins run will be failed but in aetos it will be all green